MinimedKit/PumpEvents/DailyTotal523PumpEvent.swift

```diff
@@ -33,7 +33,7 @@ public struct Sara6EPumpEvent: PumpEvent {
     
     public var dictionaryRepresentation: [String: AnyObject] {
         return [
-            "_type": "Sara6E",
+            "_type": "DailyTotal523",
```

Comment:
On Thu, Sep 08, 2016 at 06:22:01PM -0700, Nathan Racklyeft wrote:
> Do you have a recent carelink pull you wouldn't mind contributing to that
> effort, possibly for the unit test case below?

That's odd: I have Carelink CSV files and corresponding history page
dumps for some periods where those records occur, but the CSV files
don't list them at all.  It just goes from the previous
BasalProfileStart, to the ResultDailyTotal, to the 00:00
BasalProfileStart.  The history page dumps have them though, right
before the ResultDailyTotal record.

--
Eric Cooper             e c c @ c m u . e d u